src/chrome/skin/icons/freebsd/hicolor/22x22/apps/chatzilla.png

```diff
@@ -0,0 +1 @@
+../../../../img/chatzilla22.png
```

Comment:
On 02/03/2015 17:24, foudfou wrote:
>
> In src/chrome/skin/icons/freebsd/hicolor/22x22/apps/chatzilla.png 
> <https://github.com/foudfou/FireTray/pull/167#discussion_r25653006>:
>
> > @@ -0,0 +1 @@
> > +../../../../img/chatzilla22.png
>
> No I meant the contrary: link the directory instead of each file. I 
> think you linked each file (may be wrong).
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/foudfou/FireTray/pull/167/files#r25653006>.
>
I copied each "linux" folder as-is.
S